.github/ISSUE_TEMPLATE/3-sve-1.0-issue.yaml

```diff
@@ -0,0 +1,95 @@
+name: "\U0001F41B SVE Issue"
```

Comment:
Do we need this? Are SVE issues not a subset of master issues at this time? would it make sense to have SVE a checkbox on master issues?

or if this is a SVE maybe just link to the master issue or commit without the entire repro and rest of query.

Reply:
I'd like people to intentionally ask for SVE here.